### PR TITLE
Emmanuel jsday2 javascript assignments

### PR DESCRIPTION
**Pull request by codewithmanny**
Attached are Images of the outputs for both javascript assignment and udemy screenshot

products

products2

sampleform
udemycompleted

### CODE REVIEW

File: emmanuelagyei/js/variable.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// Create new variables firstName , lastName,  
+// Create new variable myNumber and assign 2 to it 
+// Connect the firstName and lastName to construct the variable fullName, but with a space (" ") in between the first and last name. 
+// Multiply the variable myNumber by 2 and put the result in total. 
+let y =2 
+let firstName = "Emmanuel";
+let lastName = "Agyei"
+
+let fullName = firstName +  lastName;
+
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:12]: for fullname, you need to add the space (" ") between the first and last names.

File: emmanuelagyei/js/variable.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+console.log(fullName);
+
+console.log(fullName + y);
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:15]: This should output y multiplied by 2

File: emmanuelagyei/js/variable.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+// Write a JavaScript program to find a value which is nearest to 100 from two different given integer values. 
+// Example Test cases:  
+// findNearestValueTo100(80, 99) should result in 99 
+// findNearestValueTo100(-10, -9) should result in -9 
+
+function nearest (a,b){
+let a1 = 100-a 
+let b1 = 100-b
+
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:26]: You are trying to get the nearest to 100, and you have to factor in a negative 'distance'.
You should use the Math library - Math.abs(value) to get the absolute difference between the number and 100.

File: emmanuelagyei/js/variable.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+if (a1>b1){
+    return b + "is nearest"
+}else {
+    return a + "is nearest"
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:30]: This is true if a1 is less than b2.  But what if a1 and b1 are equal, then both numbers are equidistant from 100.

File: emmanuelagyei/Js-day2/products.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Products</h1>
+
+<ul>  
+    <li>Product 1  <input type="text" name="" id="" value="Add to wishlist"> </li>
+    <li>Product 2   <input type="text" name="" id="" value="Add to wishlist"> </li>
+    <li>Product 3 <input type="text" name="" id="" value="Add to wishlist"> </li>
```
Comment (rbongiorno-ey) [inline emmanuelagyei/Js-day2/products.html:15]: The inputs should be buttons, that when clicked on, will add "Product #" to the Wishlist.

File: emmanuelagyei/Js-day2/products.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+</ul>
+
+<h1>Wishlist</h1>
+<form>
+    <input type="text" name="" id="" value="Add to wishlist">
```
Comment (rbongiorno-ey) [inline emmanuelagyei/Js-day2/products.html:20]: This should be a display area which contains the Wishlist which is built out as you click the product add buttons above.

**Review by rbongiorno-ey — CHANGES_REQUESTED** [review]

**Review by rbongiorno-ey — APPROVED** [review]
Approve, assuming the variables.js from PR#119 will be the final version.

File: emmanuelagyei/Js-day2/products.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+<div>
+<h1>Wishlist</h1>
+ <ul id="wish"></ul>
+</div>
+
+<script> 
+ for (let i =1; i<4; i ++){
+    document.getElementById(i) .addEventListener("click", function() {
+        let li = document.createElement('li');
```
Comment (madhanramu) [inline emmanuelagyei/Js-day2/products.html:26]: If you add the item to Wishlist , the should be remove from products

File: emmanuelagyei/Js-day2/products.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    let products = ["Product 1", "Product 2", "Product 3"]; 
+    let wishlist = []; 
+                
+    let plist = document.getElementById("productList"); 
+    let wlist = document.getElementById("wishList");
+
+     products.forEach((item) => {
+                   
+     let li = document.createElement("li"); 
+                li.innerText = item;
```
Comment (madhanramu) [inline emmanuelagyei/Js-day2/products.html:27]: Code Indentation. Indent the code properly.

**Review by madhanramu — APPROVED** [review]

**Pull request merged**